src/mixins/name-request-mixin.ts

```diff
@@ -62,7 +63,7 @@ export default class NameRequestMixin extends Vue {
    * */
   isNrValid (nr: any): boolean {
     const requestActionCodeList = this.getResource.changeData?.nameRequestTypes ||
-      [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION]
+      [NrRequestActionCodes.CHANGE_NAME, NrRequestActionCodes.CONVERSION, NrRequestActionCodes.RESTORE]
```

Comment:
This is a fallback list, but in the resource files we declare nameRequestTypes, so this isn't really used.